src/components/edit-post/editpost.html

```diff
@@ -0,0 +1,194 @@
+<div>
+  <headermenu/>
```

Comment:
Add dynamic title here

Reply:
I didn't getting you, can you elaborate?

Reply:
what is this PR for ??
https://github.com/ldeveloper7/vue-start/pull/44

Reply:
That PR is generated by other developer

Reply:
So ??

Reply:
is it not part of your project ??

Reply:
Are you not checking the code of other developer ??

Reply:
Are you not checking, what the new thing is done by other developer ??

Reply:
if other developer has implemented the functionality of dynamic title as per page,
And you are adding new page, than you must have to add that dynamic title thing for your page..

Reply:
if you are working with team, than you must have to track what the things are going in your project

Reply:
Now just set dynamic title for your edit article page...
Make it fast..

Reply:
Ok

Reply:
:+1:

Reply:
Your branch is not update with master branch

Reply:
:+1: